lib/hanami/utils/kernel.rb

```diff
@@ -25,6 +25,14 @@ module Kernel
       # @see Hanami::Utils::Kernel.Integer
       NUMERIC_MATCHER = /\A([\d\/\.\+iE]+|NaN|Infinity)\z/.freeze
 
+      # Default precision for bigdecimal with rational number
+      #
+      # @since x.x.x
+      # @api private
+      #
+      # @see Hanami::Utils::Kernel.BigDecimal
+      BIGDECIMAL_DEFAULT_PRECISION = 1
```

Comment:
@AlfonsoUceda The only thing I'm not sure about is the value of this constant. By using this low precision, we drop accuracy, and accuracy is what you look for when using BigDecimal. Shall we consider a bigger value for this? Eg. `16`.